examples/js/controls/OrbitControls.js

```diff
@@ -679,7 +682,7 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 			case scope.mouseButtons.LEFT:
 
-				if ( event.ctrlKey || event.metaKey ) {
+				if ( scope.panKeys.some( key => event[ key ] ) ) {
```

Comment:
Please don't use arrow functions. In general, it's better to define the function once and reuse it in the `some` method call. See the following example code https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some